collections/nemo_nlp/nemo_nlp/modules/losses.py

```diff
@@ -120,12 +120,16 @@ def __init__(self, num_classes, **kwargs):
         self._criterion = nn.CrossEntropyLoss()
         self.num_classes = num_classes
 
-    def _loss_function(self, logits, labels, input_mask):
-        active_loss = input_mask.view(-1) > 0.5
+    def _loss_function(self, logits, labels, loss_mask):
+        active_loss = loss_mask.view(-1)
         active_logits = logits.view(-1, self.num_classes)[active_loss]
         active_labels = labels.view(-1)[active_loss]
 
-        loss = self._criterion(active_logits, active_labels)
+        # To support empty active_labels
```

Comment:
can you give examples of scenarios in which we might have empty active_labels?

Reply:
no, removed the empty label support